src/components/Login/Login.component.jsx

```diff
@@ -1,29 +1,50 @@
-import React from 'react';
+import React, { useRef, useContext } from 'react';
+import { StoreContext } from '../../utils/store/StoreContext';
+import { useAuth } from '../../utils/hooks/useAuth';
 import Modal from '../Layout/Modal';
-
 import classes from './Login.module.css';
 
-const Login = ({ onClose }) => {
-  const submitForm = (evt) => {
+const Login = () => {
+  const userInput = useRef();
+  const passwordInput = useRef();
+  const { logIn } = useAuth();
+  const { store, dispatch } = useContext(StoreContext);
+  const { loginModalIsOpen } = store;
+
+  const login = (evt) => {
     evt.preventDefault();
-    console.log('tomalo');
+
+    const user = userInput.current.value;
+    const password = passwordInput.current.value;
```

Comment:
Maybe in the future you can create a custom hook that handles the values of a form